test/cases/testdata/rendertemplate/rendertemplate.yaml

```diff
@@ -0,0 +1,37 @@
+cases:
+- note: rendertemplate/simple
+  query: data.test.p = x
+  modules:
+  - |
+    package test
+
+    template_string = `{{.test}}`
+    template_vars = {`test`: `hello world`}
```

Comment:
@RDVasavada the value does not have to be string, correct?

Reply:
That's correct